### ISSUE TITLE
t.web.http.handleStatus documentation update

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2930]
|<img alt="zectbumo's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| zectbumo reported|
|-|-|
|Trac ID|trac#2930|
|Type|enhancement|
|Created|2007-11-30 00:51:31Z|

twisted.web.http.{handleStatus,handleHeader,handleEndHeaders} don't show up in the API docs since the functions were assigned to a lambda.  I also corrected some other documentation to better fit the twisted documentation standard

Attachments:

* [http-doc.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/39f/39fb523d32b5c2ae9ba38a9ee0d9e9f182676c4f/19ed37c860893d24388e23c3b7e48ca311425cc5.diff) (16004 bytes) - added by zectbumo on 2007-11-30 00:53:42Z - http documentation diff
* [server-doc.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/39f/39fb523d32b5c2ae9ba38a9ee0d9e9f182676c4f/f08226544b92019225065daf4df02288f8c7f77c.diff) (4428 bytes) - added by zectbumo on 2007-11-30 00:55:54Z - server documentation diff
* [handlestatus-doc-2930.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/39f/39fb523d32b5c2ae9ba38a9ee0d9e9f182676c4f/edee041b4255e7c8095c2013d0a7ed2bd53b2f73.patch) (1202 bytes) - added by thijs on 2008-07-11 18:52:34Z - patch against r24259

<details><summary>Searchable metadata</summary>

```
trac-id__2930 2930
type__enhancement enhancement
reporter__zectbumo zectbumo
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__documentation documentation
time__1196383891000000 1196383891000000
changetime__1215802556000000 1215802556000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="zectbumo's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|zectbumo commented|
|-|-|

the server-doc.diff file I accidently started formatting to doc standard thinking it was http.py, so I just uploaded the diff thinking it couldn't hurt.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra removed owner|
|-|-|

I can update the patch for the trunk and apply it if someone reviews the docstrings, looks good to me.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve set owner to @thijstriemstra|
|-|-|

Hum I don't see why we should escape quotes, it looks useless to me. The handleStatus/handleHeader/handleEndHeaders looks good though.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set status to `assigned`|
|-|-|

Replying to [therve](#note_4):
> Hum I don't see why we should escape quotes, it looks useless to me. The handleStatus/handleHeader/handleEndHeaders looks good though.

I'll apply the handleStatus/handleHeader/handleEndHeaders part.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [24260]) Apply handlestatus-doc-2930.patch: twisted.web.http.{handleStatus,handleHeader,handleEndHeaders} show up in the API docs now.

Author: zectbumo
Reviewer: therve, thijs
Fixes: [#2930](https://github.com/twisted/twisted/issues/2930)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
